azurerm/resource_arm_recovery_services_protection_policy_vm.go

```diff
@@ -55,6 +55,12 @@ func resourceArmRecoveryServicesProtectionPolicyVm() *schema.Resource {
 				),
 			},
 
+			"timezone": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Default:  "UTC",
```

Comment:
It would be nice, however I don't know what of the many different timezone formats is valid here (there are at least 2 different sets of valid values it could be, maybe more)